cdap-ui/app/cdap/components/Experiments/DetailedView/ModelsTable/index.js

```diff
@@ -129,16 +132,80 @@ const renderMetrics = (newHeaders, model) => {
   return metrics.map(t => wrapContentWithTitleAttr(model.evaluationMetrics[t.property] || '--'));
 };
 
-const renderModelDetails = (model, newlyTrainingModel) => {
+const renderFeaturesTable = (features) => {
+  return (
+    <div className="features-grid-wrapper">
+      <div className="grid grid-container">
+        <div className="grid-header">
+          <div className="grid-item">
+            <strong> Features </strong>
+          </div>
+        </div>
+        <div className="grid-body">
+          {features.map(feature => (<div className="grid-item"> {feature}</div>))}
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const renderDirectivesTables = (directives) => {
+  return (
+    <div className="features-grid-wrapper">
+      <div className="grid grid-container">
+        <div className="grid-header">
+          <div className="grid-item">
+            <strong> Directives </strong>
+          </div>
+        </div>
+        <div className="grid-body">
+          {directives.map(directive => (<div className="grid-item"> {directive}</div>))}
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const constructModelTrainingLogs = (model, experimentId) => {
+  let splitId = objectQuery(model, 'splitDetails', 'id');
+  let nowInSeconds = Math.floor(Date.now() / 1000);
+  let startTime = nowInSeconds - ONE_WEEK_SECONDS;
+  let endTime = nowInSeconds;
+  if (splitId) {
+    let {routerServerUrl, routerServerPort} = window.CDAP_CONFIG.cdap;
+    let protocol = window.CDAP_CONFIG.sslEnabled ? 'https' : 'http';
+    if (routerServerUrl === '127.0.0.1') {
+      routerServerUrl = 'localhost';
+    }
+    let hostPort = `${protocol}://${routerServerUrl}:${routerServerPort}`;
+    let baseUrl = `/v3/namespaces/${getCurrentNamespace()}/apps/ModelManagementApp/spark/ModelManagerService/logs`;
+    let queryParams = `?filter=MDC:experiment="${experimentId}"%20AND%20MDC:model=${model.id}&start=${startTime}&end=${endTime}`;
```

Comment:
is it possible to just use `encodeURI`? would that mess up the experimentID AND MDC?